src/main/java/com/hyperwallet/clientsdk/model/DocumentVerificationDocumentsRepresentation.java

```diff
@@ -0,0 +1,120 @@
+package com.hyperwallet.clientsdk.model;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.hyperwallet.clientsdk.util.HyperwalletJsonConfiguration;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@JsonFilter(HyperwalletJsonConfiguration.INCLUSION_FILTER)
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class DocumentVerificationDocumentsRepresentation {
```

Comment:
Class name can be changed to HyperwalletDocument with more details about the class in the javadoc

Reply:
Changed class name.